plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/PingNuageVspCommand.java

```diff
@@ -17,18 +17,37 @@
 // under the License.
 //
 
-package com.cloud.agent.api.element;
+package com.cloud.agent.api;
 
-import com.cloud.agent.api.Answer;
+import com.cloud.host.Host;
 
-public class ShutDownVpcVspAnswer extends Answer {
+public class PingNuageVspCommand extends PingCommand {
 
-    public ShutDownVpcVspAnswer(ShutDownVpcVspCommand cmd, boolean success, String details) {
-        super(cmd, success, details);
+    private final boolean shouldAudit;
+
+    public PingNuageVspCommand(Host.Type type, long id, boolean shouldAudit) {
+        super(type, id);
+        this.shouldAudit = shouldAudit;
     }
 
-    public ShutDownVpcVspAnswer(ShutDownVpcVspCommand cmd, Exception e) {
-        super(cmd, e);
+    public boolean shouldAudit() {
+        return shouldAudit;
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
This equals implementation should include attributes from the ``PingCommand`` super class.